spec/Section 4 -- Introspection.md

```diff
@@ -289,44 +312,44 @@ Fields
 * All other fields must return {null}.
 
 
-#### Interface
+**Interface**
 
 Interfaces are an abstract type where there are common fields declared. Any type
 that implements an interface must define all the fields with names and types
 exactly matching. The implementations of this interface are explicitly listed
 out in `possibleTypes`.
 
-Fields
+Fields\:
 
 * `kind` must return `__TypeKind.INTERFACE`.
 * `name` must return a String.
 * `description` may return a String or {null}.
-* `fields`: The set of fields required by this interface.
+* `fields` must return the set of fields required by this interface.
   * Accepts the argument `includeDeprecated` which defaults to {false}. If
     {true}, deprecated fields are also returned.
-* `interfaces`: The set of interfaces that this interface implements.
+* `interfaces` must return the set of interfaces that this interface implements.
 * `possibleTypes` returns the list of types that implement this interface.
   They must be object types.
 * All other fields must return {null}.
 
 
-#### Enum
+**Enum**
 
 Enums are special scalars that can only have a defined set of values.
 
-Fields
+Fields\:
 
 * `kind` must return `__TypeKind.ENUM`.
 * `name` must return a String.
 * `description` may return a String or {null}.
-* `enumValues`: The list of `EnumValue`. There must be at least one and they
-  must have unique names.
+* `enumValues` must return the set of `EnumValue`. There must be at least one
+  and they must have unique names.
```

Comment:
`EnumValue` -> `__EnumValue` and edited for consistency with the text for input object `inputFields`:

```suggestion
* `enumValues` must return the set of enum values as a list of `__EnumValue`. There must be at least one
  and they must have unique names.
```